eng/pipelines/azure-pipelines.yml

```diff
@@ -5,6 +5,20 @@ trigger:
     - main
     - release/*
     - internal/release/*
+  paths:
```

Comment:
Why are we adding these exclusions? These don't seem related to the migration.

Reply:
Indeed, it's not directly related to the migration. It's an optimisation I copied from dotnet/extensions pipelines.